docs/spark-configuration.md

```diff
@@ -189,13 +189,13 @@ df.write
 | write-format           | Table write.format.default | File format to use for this write operation; parquet, avro, or orc |
 | target-file-size-bytes | As per table property      | Overrides this table's write.target-file-size-bytes          |
 | check-nullability      | true                       | Sets the nullable check on fields                            |
-| snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary  |
+| snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary (the `snapshot-property.` prefix is only required for DSv2 and not for properties passed through `CommitMetadata`)  |
```

Comment:
I don't think it is necessary to mention CommitMetadata here. Long descriptions make these tables much harder to read.

Reply:
fixed